airbyte-commons-worker/src/main/java/io/airbyte/workers/internal/DefaultAirbyteSource.java

```diff
@@ -61,23 +60,19 @@ public class DefaultAirbyteSource implements AirbyteSource {
   private Integer exitValue = null;
   private final boolean featureFlagLogConnectorMsgs;
 
-  public DefaultAirbyteSource(final IntegrationLauncher integrationLauncher, final FeatureFlags featureFlags) {
-    this(integrationLauncher, new DefaultAirbyteStreamFactory(CONTAINER_LOG_MDC_BUILDER), new DefaultProtocolSerializer(), featureFlags);
+  public DefaultAirbyteSource(final IntegrationLauncher integrationLauncher,
```

Comment:
It might be easier to leave a copy of the old constructor. With the monorepo, I think we'll end up duplicating the `DefaultAirbyteSource` for the connector tests and eventually reimplementing a specific one. This could limit the scope of this PR.

Reply:
Ok, but it will need to have an heartbeatMonitor still. I'm not sure why keeping the old ctor will help/